azure-cli/monitor/alert.utilities.ps1

```diff
@@ -0,0 +1,259 @@
+. "$PSScriptRoot\..\utilities\deploy.utilities.ps1"
```

Comment:
Maybe we should consider to add some documentation to each method, to describe what the purpose is, and what is being returned - if not obvious.

Reply:
Removed a couple of unused functions, the rest should be clear from the names.